templates/slim/helpers.rb

```diff
@@ -5,10 +5,6 @@
 
   # Needed only in compile-time.
   require 'slim-htag' if defined? Slim
```

Comment:
You can remove this as well, it doesn’t work as intended anyway.

Reply:
done